ui/lib/core/addon/utils/advanced-secret.js

```diff
@@ -31,7 +31,14 @@ export function obfuscateData(obj) {
     if (Array.isArray(obj[key])) {
       newObj[key] = obj[key].map(() => '********');
     } else if (typeof obj[key] === 'object') {
-      newObj[key] = obfuscateData(obj[key]);
+      // unfortunately in javascript typeof null returns object
+      // this is due to a "historical js bug that will never be fixed"
+      // we handle this situation here
+      if (obj[key] === null) {
```

Comment:
Could we instead move this to its own `else if` block before the 'object' check? that would flatten the logic